packages/components/filters/src/filters.stories.tsx

```diff
@@ -1,34 +1,353 @@
-import Filters, { type TFiltersProps } from './filters';
+import { useState } from 'react';
 import type { Meta, StoryFn } from '@storybook/react';
-import { designTokens } from '@commercetools-uikit/design-system';
+import PrimaryButton from '@commercetools-uikit/primary-button';
+import Filters, { type TFiltersProps } from './filters';
+import {
+  SearchInputComponent,
+  FruitsRadioInput,
+  ColorNameTextInput,
+  SecondaryColorsInput,
+  PrimaryColorsInput,
+  PrimaryColorsRadioInput,
+  PrimaryColorsTextInput,
+  OperatorsInput,
+  SelectedValueWithOperator,
+} from './fixtures/inputs';
+import {
+  FILTER_GROUP_KEYS,
+  FILTER_GROUPS,
+  FRUIT_OPTIONS,
+  OPERATOR_OPTIONS,
+} from './fixtures/constants';
+
+type TFiltersPropsWithCustomArgs = TFiltersProps & {
+  label?: string;
+  renderMenuBody?: string;
+  renderOperatorsInput?: boolean;
+  renderApplyButton?: boolean;
+  isDisabled?: boolean;
+  isPersistent?: boolean;
+};
 
-const meta: Meta<typeof Filters> = {
+const CustomSearchExample = () => <div>im an example</div>;
+
+const meta: Meta<TFiltersPropsWithCustomArgs> = {
   title: 'components/Filters',
   component: Filters,
-  // tags: ['local-dev'],
-  argTypes: {},
+  args: {
+    renderSearchComponent: <SearchInputComponent />,
+    label: 'Primary Colors',
+    renderOperatorsInput: true,
+    renderMenuBody: 'select',
+    renderApplyButton: false,
+    isDisabled: false,
+    isPersistent: true,
+    defaultOpen: true,
+  },
+  argTypes: {
+    appliedFilters: {
+      control: false,
```

Comment:
I don't think it's worth giving the user controls over this, since the json editor is flaky and hard to use, and the strorybook component handles this anyway